### PR TITLE
resolve-issue-1051

### PR DESCRIPTION
**Pull request by rms1000watt**
Resolve issue https://github.com/kubernetes-sigs/aws-alb-ingress-controller/issues/1051

### CODE REVIEW

**Comment by rms1000watt** [comment]
Just signed CLA

**Comment by M00nF1sh** [comment]
@rms1000watt 
/ok-to-test
Thanks for fixing this :D, i'll merge this in(but it will be overwritten and also fixed by https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/1088 soon)

File: internal/alb/ls/rules.go [hunk 1]
```diff
@@ -305,6 +305,9 @@ func rulesChangeSets(current, desired []elbv2.Rule) (add []elbv2.Rule, modify []
 func condition(field string, values ...string) *elbv2.RuleCondition {
 	return &elbv2.RuleCondition{
 		Field:  aws.String(field),
+		PathPatternConfig: *elbv2.PathPatternConditionConfig{
```
Comment (alejandrox1) [inline internal/alb/ls/rules.go]:
```suggestion
		PathPatternConfig: &elbv2.PathPatternConditionConfig{
```

looking at the logs from ci https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_aws-alb-ingress-controller/1093/pull-aws-alb-ingress-controller-e2e-test/1202761243954253825#1:build-log.txt%3A244

```
internal/alb/ls/rules.go:308:22: invalid indirect of elbv2.PathPatternConditionConfig literal (type elbv2.PathPatternConditionConfig)
```

**Comment by rms1000watt** [comment]
thank you @alejandrox1

**Comment by M00nF1sh** [comment]
Hi @rms1000watt 
I'm going to close this one, given i merged the https://github.com/kubernetes-sigs/aws-alb-ingress-controller/pull/1088
Thanks a lot for the contributing, and i should have merged this sooner

**Pull request closed without merging**